src/schemas/Helpdesk/gqlTypeDefs.ts

```diff
@@ -0,0 +1,23 @@
+import { gql } from 'apollo-server-express';
+
+export default gql`
+  type TicketCreationResponse {
+    requestOk: Boolean!
+    statusText: String!
+  }
+
+  type Mutation {
+    createJiraTicket(
+      dropdownValue: String!
```

Comment:
`dropdownValue` doesn't really mean much when looking at the schema. Knowing what it is though, something like `messageCategory` or even `summaryPrefix` seems to make more sense. 

Another note on this is, since it's a limited list of things, this might make sense as an enum.

Reply:
I was thinking of doing the enum, but thought it would have to be defined in the UI end of things anyway, and so wouldn't adding it here as well required twice the maintenance, should something change?

Reply:
Since this mutation is publicly available, the UI is not guaranteed to be the only place requests are coming from, it's a good vandalism mitigation measure :) also prevent us making mistakes if we need to call this from other places too